setup.cfg

```diff
@@ -32,5 +32,7 @@ tag_prefix = v
 
 [mypy]
 exclude=openff/utilities/_tests
-warn_unused_configs = True
-warn_unused_ignores = True
+python_version=3.12
```

Comment:
(not blocking) Not sure if intentional, but if it is and comments are allowed in this file, it could be good to explain why this is pinned and when it can be removed.